index.md

```diff
@@ -1,57 +1,198 @@
-	<p>All the code is available to you ensuring you'll have the ability to make changes when you need to or even port to whole new platforms.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Community</h4>
-	<p>With 1000s of shipped games we have built up a vibrant community of developers which use MonoGame for both fun and profit.</p>
-</div>
+<section class="hero" >
+	<div id="hero-background-image" />
+	<div class="hero-content">
+		<p class="slogan">
+			One framework for creating<br/>
+			powerful cross-platform games
+		<div class="hero-button-container">
+			<a type="button" class="btn btn-lrg btn-monogame" href="/articles/getting_started/index.html">
+				Getting Started <i class="bi bi-arrow-right"></i>
+			</a>
+			<a type="button" class="btn btn-lrg btn-monogame" href="https://github.com/MonoGame/MonoGame">
+				Fork Us <i class="bi bi-github"></i>
+			</a>
+		</div>
+	</div>
+	<div class="hero-credits">
+		<a href=""><img id="credits-logo" src=""></a><br/>
+		<a id="credits-built-with" href="https://twitter.com/search?q=BuiltWithMonoGame">#BuiltWithMonoGame</a>
+	</div>
 </section>
-</div>
+
+<section class="container-xxl my-5">
+	<div class="row">
+		<div class="col-12 col-md-6">
+			<h3 class="fw-semibold mb-4 text-body-emphasis">Managed Code</h3>
+			<p class="lead">
+				By leveraging C# and other .NET languages on Microsoft and Mono platforms you can write modern, fast, and reliable game code using your editor of choice.
+		   </p>
+		   <ul class="list-unstyled">
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Visual Studio (recommended)</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Visual Studio Code</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> JetBrains Rider</li>
+		   </ul>
+		</div>
+		<div class="col-12 col-md-6 d-flex flex-row justify-content-center">
+			<img src="/images/home-page/managed-code.png" class="featurette-image" loading="lazy">
+		</div>
+	</div>
+</section>
+
+
+
+<section class="container-xxl my-5">
+	<div class="row">
+		<div class="col-12 col-md-6">
+			<div class="d-flex flex-row flex-wrap gap-5">
+				<img id="windows-logo" class="cross-platform-logo" loading="lazy">
+				<img id="apple-logo" class="cross-platform-logo" loading="lazy">
+				<img id="linux-logo" class="cross-platform-logo" loading="lazy">
+				<img id="android-logo" class="cross-platform-logo" loading="lazy">
+				<img id="ios-logo" class="cross-platform-logo" loading="lazy">
+				<img id="playstation-logo" class="cross-platform-logo" loading="lazy">
+				<img id="xbox-one-logo" class="cross-platform-logo" loading="lazy">
+				<img id="switch-logo" class="cross-platform-logo" loading="lazy">
+			</div>
+		</div>
+		<div class="col-12 col-md-6">
+			<h3 class="fw-semibold mb-4 text-body-emphasis">Cross-Platform</h3>

```

Comment:
There isn't much of a gap between these two sections when on mobile:

![image](https://github.com/MonoGame/monogame.github.io/assets/103522/93090361-6fec-4ced-a3e7-085b57982b53)

Reply:
Bottom margin (`md-5`) added after icons https://github.com/MonoGame/monogame.github.io/pull/19/commits/e92a802a2f95790cfec454145b2fe0697fbb5d65